.github/workflows/scripts/start-acs.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/bash
+set -euox pipefail
+
+
+cd $STACKROX_DIR
+
+echo "GITHUB_OUTPUT= ${GITHUB_OUTPUT}"
+./deploy/k8s/central.sh
+kubectl -n stackrox port-forward deploy/central 8000:8443 > /dev/null 2>&1 &
+sleep 20
+
+./deploy/k8s/sensor.sh
+
+kubectl -n stackrox set env deploy/sensor MUTEX_WATCHDOG_TIMEOUT_SECS=0 ROX_FAKE_KUBERNETES_WORKLOAD=long-running ROX_FAKE_WORKLOAD_STORAGE=/var/cache/stackrox/pebble.db
+kubectl -n stackrox patch deploy/sensor -p '{"spec":{"template":{"spec":{"containers":[{"name":"sensor","resources":{"requests":{"memory":"3Gi","cpu":"2"},"limits":{"memory":"12Gi","cpu":"4"}}}]}}}}'
+
+kubectl -n stackrox set env deploy/central MUTEX_WATCHDOG_TIMEOUT_SECS=0
+kubectl -n stackrox patch deploy/central -p '{"spec":{"template":{"spec":{"containers":[{"name":"central","resources":{"requests":{"memory":"3Gi","cpu":"2"},"limits":{"memory":"12Gi","cpu":"4"}}}]}}}}'
+
+max_tries=200
+tries=0
+while [ "$tries" -lt "$max_tries" ]; do
+    ready="$(kubectl -n stackrox get pod -l app=central -o jsonpath='{.items[*].status.containerStatuses[?(@.ready == true)]}')"
+
+    if [[ -n "$ready" ]]; then
+	    echo "Central is running"
+	    break
+    fi
+
+    tries=$((tries + 1))
+    sleep 1
+done
+
+sleep 60
```

Comment:
I have added this check and sleep here so that when the secured cluster tries to connect to central, it is ready.

Reply:
Is it intentional to wait for readiness and then sleep for another 60 seconds?

Reply:
I think there is also no condition to stop the script if central does not become ready.

Reply:
Yes, it is intentional to wait another 60 seconds after readiness, just to be on the safe side.

Reply:
I have removed the sleep 60 and I am using an existing function to test for central readiness.